src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx

```diff
@@ -68,9 +69,6 @@ export default function OpeningPeriod({
       className="opening-period"
       data-test={`openingPeriod-${datePeriod.id}`}>
       <div className="opening-period-header">
-        <div className="opening-period-dates opening-period-header-column">
-          <div>{formattedDateRange}</div>
-        </div>
         <div className="opening-period-title opening-period-header-column">
           {datePeriodName ? (
             <h4>{datePeriodName}</h4>
```

Comment:
This title now seems too small when there are even and odd week titles: 
<img width="1303" alt="Screenshot 2022-06-15 at 14 20 30" src="https://user-images.githubusercontent.com/1610860/173815402-edbe6142-79c9-4ccd-a6fa-97e5e4c13b56.png">

Reply:
Fixed